### PR TITLE
CTCP-5537: Update README

### PR DESCRIPTION
**Pull request by Reece-Carruthers**

### CODE REVIEW

**Comment by Reece-Carruthers** [comment]
Formatted README.md can be viewed [here](https://github.com/hmrc/ctc-departure-trader-details-frontend/blob/06ad8753d296e95696c6ea3f949b7c52ab46d056/README.md)

File: README.md [hunk 1]
```diff
@@ -21,17 +18,114 @@ To toggle between the Phase 5 transition and post-transition modes we have defin
 #### Transition
 <pre>
 sm2 --start CTC_TRADERS_P5_ACCEPTANCE_TRANSITION
-sm2 --stop CTC_DEPARTURE_TRADER_DETAILS_FRONTEND_TRANSITION
+sm2 --stop MANAGE_TRANSIT_MOVEMENTS_DEPARTURE_FRONTEND_TRANSITION
```
Comment (jackheslop96) [inline README.md:21]: Can we leave this as the relevant service

File: README.md [hunk 2]
```diff
@@ -21,17 +18,114 @@ To toggle between the Phase 5 transition and post-transition modes we have defin
...
 sbt -Dplay.additional.module=config.TransitionModule run
 </pre>
 
 #### Final
 <pre>
 sm2 --start CTC_TRADERS_P5_ACCEPTANCE
-sm2 --stop CTC_DEPARTURE_TRADER_DETAILS_FRONTEND
+sm2 --stop MANAGE_TRANSIT_MOVEMENTS_DEPARTURE_FRONTEND
```
Comment (jackheslop96) [inline README.md:28]: Can we leave this as the relevant service

Comment (Reece-Carruthers) [reply README.md:28]: Yep, I missed that when copying over previous changes

File: README.md [hunk 3]
```diff
@@ -21,17 +18,114 @@ To toggle between the Phase 5 transition and post-transition modes we have defin
...
+| `banners.showUserResearch`   | `Boolean`     | `sbt -Dbanners.showUserResearch=true run`                       | Controls whether or not we show the user research banner.                                                                                                                                      |
+| `play.additional.module`     | `String`      | `sbt -Dplay.additional.module=config.PostTransitionModule run`  | Controls which module (TransitionModule or PostTransitionModule) we bind to the application at start-up.                                                                                       |
+| `play.http.router`           | `String`      | `sbt -Dplay.http.router=testOnlyDoNotUseInAppConf.Routes run`   | Controls which router is used for the application, either `prod.Routes` or `testOnlyDoNotUseInAppConf.Routes`                                                                                  |
+
+### Running Scaffold
+
+Create New Page in sbt:
```
Comment (jackheslop96) [inline README.md:45]: For the departure sub microservices I'd be tempted to just have a link to the manage-transit-movements-departure-frontend README

Comment (Reece-Carruthers) [reply README.md]: Yeah, that will make it easier to manage if any of that information does chance in the future, pushed an update

**Comment by Reece-Carruthers** [comment]
Updated README.md can be viewed [here](https://github.com/hmrc/ctc-departure-trader-details-frontend/blob/6abebd85aa7eafad4ce243b23d08d7e7ef1208fc/README.md)

**Review by jackheslop96 — APPROVED** [review]

**Pull request merged**